### PR TITLE
Niamh25/DOC-5363/Update-CSDE-templates-to-use-Digicert-for-GA

### PR DESCRIPTION
**Pull request by Niamh25**
Updated CSDE images to use Digicert for GA

### CODE REVIEW

**Review by yarivus — APPROVED** [review]
LGTM

File: content/en/platform/corda/5.0-beta/introduction/introduction.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
+---
```
Comment (nadinequinr3) [inline content/en/platform/corda/5.0-beta/introduction/introduction.md:1]: I think this is an error @Niamh25? We don't want Beta content in the GA branch.

**Review by nadinequinr3 — CHANGES_REQUESTED** [review]

File: content/en/platform/corda/5.0-beta/introduction/introduction.md (hunk 1 above)
Comment (Niamh25) [reply content/en/platform/corda/5.0-beta/introduction/introduction.md:1]: Yep this is an error, I removing it now!

**Review by nadinequinr3 — APPROVED** [review]

**Review by yarivus — COMMENTED** [review]
thx @Niamh25

**Pull request merged**